getting-started/README.md

```diff
@@ -0,0 +1,111 @@
+# Get started with ONNX Runtime Training
+
+Train a PyTorch transformer model with ONNX Runtime.
+
+This sample trains the model defined here: https://pytorch.org/tutorials/beginner/transformer_tutorial.html. This is a transformer model composed of PyTorch transformer components such as the PositionalEncoder, TransformerEncoder and word Embedding.
+
+Trained using the Wikitext-2 dataset (from `torchtext.dataset`), the language model assigns a probability for the likelihood of a given word (or a sequence of words) to follow a sequence of words.
+
+This sample uses the base docker image `mcr.microsoft.com/azureml/onnxruntime-training:0.1-rc2-openmpi4.0-cuda10.2-cudnn7.6-nccl2.7.6` to train the model.
+
+The purpose of this sample is to demonstrate the concepts and usage of ONNX Runtime Training, rather than to show its performance capabilities. For more complex samples that do demonstrate improved training performance at scale, please refer to the other samples in this repo:
+
+* [NVIDIA BERT pre-training](../nvidia-bert)
+* [HuggingFace GPT-2 fine-tuning](../huggingface-gpt2)
+
+All instructions in this sample assume you have cloned this repo and are located in the root directory of the getting-started sample.
+
+```bash
+git clone https://github.com/microsoft/onnxruntime-training-examples
+cd getting-started
+```
+
+## Code walk through
+
+The training code is based on the original PyTorch training code, which is provided for reference in the `train.py` file.
+
+### Model
+
+The PyTorch model is defined in `model.py`.
+
+### Trainer class
+
+See `train_ort.py` for the ONNX Runtime Training code. In its `train_step` method, the ORTTrainer encapsulates one pass (forward pass, loss calculation, backward pass, and optimizer weight adjustment) of the training loop.
```

Comment:
do we need a disclaimer here about the coming changes to the API?

Reply:
I'll update the sample shortly with the new API